security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -17,6 +17,62 @@ To access the latest release, see xref:../../security/compliance_operator/co-man
+
+* For previous releases of Compliance Operator, the scans would generate an error log for the reconcile loop on the operator pod. With this release, the Compliance Operator controller logic is more stable. (link:https://issues.redhat.com/browse/OCPBUGS-51267[*OCPBUGS-51267*])
+
+* Previously, the rule `file-integrity-exists` would fail during CI runs on `aarch64` clusters. Now that `aarch64` clusters are supported, the rule failure has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-52884[*OCPBUGS-52884*])
+
+* Previously, the rule `file-integrity-notification-enabled` would fail during CI runs on `aarch64` clusters. Now that `aarch64` clusters are supported, this rule failure has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-52885[*OCPBUGS-52885*])
+
+* Before this release of the Compliance Operator, the rule `kubelet-configure-tls-cipher-suites` was failing in CI tests for the API server ciphers, resulting in `E2E-FAILURE` status. The rule was not recognizing new ciphers from RFC 8446. This has been resolved and the rule is now being evaluated correctly. (link:https://issues.redhat.com/browse/OCPBUGS-54212[*OCPBUGS-54212*])
+
+* Previously, the Compliance Operator platform scan would fail and produce the message `failed to parse Ignition config`. With this release, this fault has been resolved and the scan returns either `Compliant` or `Non-Compliant`. link:https://issues.redhat.com/browse/OCPBUGS-54403[*OCPBUGS-54403*])
+
+* Before this release of Compliance Operator, the rule `directory-access-var-log-audit` appears to fail in FedRAMP moderate and high RHCOS profile scans, even after applying the remediation. With this release, the rule correctly completes and detects correct remediation. (link:https://issues.redhat.com/browse/OCPBUGS-54605[*OCPBUGS-54605*])
+
+* Before this release of Compliance Operator, several rules were not platform aware, creating unneeded errors. Now that the rules have been properly ported to other architectures, those rules run correctly. (link:https://issues.redhat.com/browse/OCPBUGS-53041[*OCPBUGS-53041*])
+
+* Before this release, all `RHCOS` profiles return `NOT-APPLICABLE` for OCP 4.19. With this release, `RHCOS` profiles return normal results. (link:https://issues.redhat.com/browse/OCPBUGS-55140[*OCPBUGS-55140*])
+
+* Previously, rule `file-groupowner-ovs-conf-db-hugetlbf` would fail unexpectedly. With this release, the rule fails only when this is the needed result. (link:http://issues.redhat.com/browse/OCPBUGS-55180[*OCPBUGS-55190*])
+
+* Previously, Compliance Operator rules did not contain correct NERC-CIP profile annotations. The incorrect annotations would provide users with erroneous output on the range of baselines such as FedRAMP High, NERC-CMP, PCI-DSS and CIS. The output has been corrected and now the results are correct as read by the customer. (link:http://issues.redhat.com/browse/OCPBUGS-55181[*OCPBUGS-55181*])
```

Comment:
This should have only been something that happened in between 1.6.2 and 1.7.0. I don't think a customer would have observed anything wrong, so if we leave it off the release notes that fine with me.